platform/mds/mds/src/main/java/org/motechproject/mds/service/impl/history/HistoryServiceImpl.java

```diff
@@ -1,12 +1,12 @@
 package org.motechproject.mds.service.impl.history;
 
 import org.motechproject.mds.domain.EntityType;
-import org.motechproject.mds.service.HistoryTrashClassHelper;
 import org.motechproject.mds.query.Property;
 import org.motechproject.mds.query.PropertyBuilder;
 import org.motechproject.mds.query.QueryParams;
 import org.motechproject.mds.query.QueryUtil;
 import org.motechproject.mds.service.HistoryService;
+import org.motechproject.mds.service.HistoryTrashClassHelper;
```

Comment:
Why is this import moved below HistoryService?

Reply:
Previously imports were in wrong order, after the change org.motechproject.mds.service.HistoryTrashClassHelper is after 
org.motechproject.mds.service.HistoryService, so now we have a proper alphabetical order